modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -4126,6 +4126,34 @@ export const console: ConsoleNS = {
                             message: "Update successful"
                         }
                     },
+                    updateSMSOTPAuthenticator: {
```

Comment:
Don't we have i18n translations for other languages?

Reply:
I didn't include them because the whole `console:develop.features.authenticationProvider.notifications` section does not exist in other languages. Should I include them? @ThaminduDilshan